core/src/Cuda/Kokkos_Cuda_Instance.hpp

```diff
@@ -450,7 +450,7 @@ class CudaInternal {
   cudaError_t cuda_func_set_attributes_wrapper(T* entry, cudaFuncAttribute attr,
                                                int value) const {
     if constexpr (setCudaDevice) set_cuda_device();
-    return cudaFuncSetAttributes(entry, attr, value);
```

Comment:
Fixed in #6786.

Reply:
does that mean I should revert this fix in my PR?

Reply:
Just rebase it on top of `develop`.

Reply:
done